### PR TITLE
Adopt cybereason pack

### PR DESCRIPTION
**Pull request by suraj-metron**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by mchasepan — CHANGES_REQUESTED** [review]
Waiting on Ishai from Cybereason to complete partner paperwork prior to merge.

**Comment by eyalpalo** [comment]
@suraj-metron any update?

**Comment by suraj-metron** [comment]
@eyalpalo Cybereason is in process of completing the partner paperwork. We will mark this Draft PR as 'Ready to Merge' after the paperwork is completed.

**Review by eyalpalo — APPROVED** [review]

**Review by eyalpalo — APPROVED** [review]

File: Packs/Cybereason/ReleaseNotes/1_0_19.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+##### Cybereason
+Note: Support for this pack will be moving to the partner around April 11th, 2022
```
Comment (mchasepan) [inline Packs/Cybereason/ReleaseNotes/1_0_19.md:2]:
```suggestion
Note: Support for this pack will be moving to the partner around May 15th, 2022
```

**Review by mchasepan — CHANGES_REQUESTED** [review]

**Review by mchasepan — APPROVED** [review]

**Pull request merged**